packages/world/ts/node/getSystemContracts.ts

```diff
@@ -22,10 +22,22 @@ export async function getSystemContracts({
     pattern: path.join(config.sourceDirectory, "**"),
   });
 
+  // Get systems from the top-level systems object
+  const topLevelSystems = Object.keys(config.systems);
+
+  // Get systems from namespaces
+  const namespaceSystems = Object.values(config.namespaces || {}).flatMap((namespace) =>
+    Object.keys(namespace.systems || {}),
```

Comment:
ideally we map these to namespace.label rather than key to keep tooling consistent

Reply:
sorry, the namespaces flatMap is the "standard" path for this kinda thing